docs/standard/serialization/system-text-json-how-to.md

```diff
@@ -182,6 +182,15 @@ To deserialize from a file by using asynchronous code, call the <xref:System.Tex
 
 :::code language="csharp" source="snippets/system-text-json-how-to/csharp/RoundtripToFileAsync.cs" id="Deserialize":::
 
+> [!TIP]
+> If you have JSON that you want to deserialize, and you don't have the class to deserialize it into, Visual Studio 2019 can automatically generate the class you need:
+>
+> 1. Copy the JSON that you need to deserialize.
+> 1. Create a class file and delete the template code.
+> 1. Choose **Edit** > **Paste Special** > **Paste JSON as Classes**.
```

Comment:
```suggestion
> 1. Copy the JSON that you need to deserialize
> 1. Create a class file and delete the template code
> 1. Choose **Edit** > **Paste Special** > **Paste JSON as Classes**
```

Reply:
Per the [style guide](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=36412), 

>In bulleted and numbered lists, end each list item with a period if:
>
> * Any item forms a complete sentence when combined with the list introduction that precedes the colon. 
Exception Don’t use periods if all items have three or fewer words or if the items are UI labels, headings, subheadings, or strings.
> * Any item by itself is a complete sentence.